README.md

```diff
@@ -50,5 +50,5 @@ By default, the `book` theme will number the chapters and pages in the left menu
 You can disable that by setting the `book_numbered_chapters` in `extra`:
```

Comment:
can you update that line as well?

Reply:
Done. Sorry should have searched first more than one instance!